pages/index.html

```diff
@@ -59,44 +60,43 @@
     <div style="float: right; margin-top: 16px; padding-right: 8px;">
         <img src="{% static '/static/images/squiggle.png' %}">
     </div>-->
-  <p style="text-align: center;">The most advanced free DJ software can be yours in seconds.</p>
+  <p style="text-align: center;">DJ your way. For free.</p>
   <div style="clear: both;"></div>
 </div> <!-- gapfiller -->
 
 <div class="content">
-  <H1>{% trans "Incredible Features, Unbeatable Value" %}</H1>
+  <h1>{% trans "Flexible Features For All DJs" %}</h1>
```

Comment:
What is a "Flexible feature"?  A Feature is never for all DJs.

"Mixxx already covers many use-cases and is open to add your own"
"Mixxx can be tweaked to your needs" 

The "High value for free" is one of the most important property of Mixxx. 

I know users sticking with Virtual DJ just because they have payed for it. 
In German we say: "Was nichts kostet ist auch nichts". We should make sure that this does not apply for Mixxx.